frappe/utils/__init__.py

```diff
@@ -664,9 +664,51 @@ def is_markdown(text):
 		return not NON_MD_HTML_PATTERN.search(text)
 
 
+class cached_prop:
```

Comment:
Should we use the default for 3.12+? :eyes:

Reply:
Ok I will implement it.